tests/drop_counters/test_drop_counters.py

```diff
@@ -147,6 +148,67 @@ def enable_counters(duthost):
             logger.info("Restoring counter '{}' state to disable".format(port))
             duthost.command("counterpoll {} disable".format(port))
 
+@pytest.fixture
+def acl_setup(duthost):
+    """ Create acl rule defined in config file. Delete rule after test case finished """
+    base_dir = os.path.dirname(os.path.realpath(__file__))
+    template_dir = os.path.join(base_dir, 'acl_templates')
+    acl_rules_template = "acltb_test_rule.json"
+    del_acl_rules_template = "acl_rule_del.json"
+    dut_tmp_dir = os.path.join("tmp", os.path.basename(base_dir))
+
+    duthost.command("mkdir -p {}".format(dut_tmp_dir))
+    dut_conf_file_path = os.path.join(dut_tmp_dir, acl_rules_template)
+    dut_clear_conf_file_path = os.path.join(dut_tmp_dir, del_acl_rules_template)
+
+    logger.info("Generating config for ACL rule, ACL table - DATAACL")
+    duthost.template(src=os.path.join(template_dir, acl_rules_template), dest=dut_conf_file_path)
+    logger.info("Generating clear config for ACL rule, ACL table - DATAACL")
+    duthost.template(src=os.path.join(template_dir, del_acl_rules_template), dest=dut_clear_conf_file_path)
+
+    logger.info("Applying {}".format(dut_conf_file_path))
+    duthost.command("config acl update full {}".format(dut_conf_file_path))
+    yield
```

Comment:
Does the code after yield execute when the test code throws an exception?
IMO the way to be sure that cleanup executes is:
```
try:
    yield
finally:
```

Reply:
Fixed.